.travis.yml

```diff
@@ -0,0 +1,10 @@
+services:
+  - docker
+
+sudo: false
+
+script:
+  - scripts/test.sh
+
+# This isn't used, but makes slightly more sense than ruby, the default.
+language: java
```

Comment:
I just think it makes more sense to run these tests in a Java environment vs. a Ruby environment, even though everything will actually happen in a Docker environment. But yeah probably better to just let it be Ruby.